### DISCUSSION TITLE
Problem flashing WROOM board

### DISCUSSION THREAD

**Discussion by mausland** [MoonModules/WLED-MM#207] [Boards]
Hi,

I'm looking forward to trying MoonModules.  When I go to flash my ESP32-WROOM-32 board with [WLEDMM_14.5.0-beta_esp32S3_WROOM-2_M.bin], it flashes and then seems to go into a loop with the following.

[08:51:38]rst:0x3 (SW_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
[08:51:38]configsip: 0, SPIWP:0xee
[08:51:38]clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
[08:51:38]mode:DOUT, clock div:1
[08:51:38]load:0x3fff0018,len:4
[08:51:38]load:0x3fff001c,len:1044
[08:51:38]load:0x40078000,len:8896
[08:51:38]load:0x40080400,len:5816
[08:51:38]entry 0x400806ac
[08:51:38]ets Jul 29 2019 12:21:46

I'm not sure what I am doing wrong...  Any help would be great.

Thanks,

Mike

**Comment by dosipod** [comment]
@mausland The bin you are trying to use is for esp32S3

**Reply by mausland** [reply]
Thanks for the reply.  Relative noob here, I saw WROOM and just assumed.  Do you have a reco for which bin to use?

**Reply by dosipod** [reply]
https://github.com/MoonModules/WLED/releases/download/v14.5.0-beta/WLEDMM_14.5.0-beta_esp32_4MB_M.bin
or 
https://github.com/MoonModules/WLED/releases/download/v14.5.0-beta/WLEDMM_14.5.0-beta_esp32_4MB_S.bin

**Reply by mausland** [reply]
Thank you!  Working!

**Reply by softhack007** [reply]
> The bin you are trying to use is for esp32S3

Correct - the "S3 wroom-2" is a very special board with ESP32-S3, 32MB flash and 16 mb PSRAM. It has nothing in common with "esp32 wroom" boards. Actually I'm a bit confused how @mausland  managed to upload a firmware for the wrong  CPU - normally esptool would complain loudly.

As @dosipod explained, any esp32 4MB firmware is good for esp32 wroom, as these boards are bread&butter i.e. nothing special.
